src/main/scala/edu/ie3/simona/agent/grid/congestion/VoltageRange.scala

```diff
@@ -39,32 +40,34 @@ import javax.measure.quantity.Dimensionless
   *   For voltage change.
   */
 final case class VoltageRange(
-    deltaPlus: ComparableQuantity[Dimensionless],
-    deltaMinus: ComparableQuantity[Dimensionless],
-    suggestion: ComparableQuantity[Dimensionless],
+    deltaPlus: squants.Dimensionless,
+    deltaMinus: squants.Dimensionless,
+    suggestion: squants.Dimensionless,
 ) {
 
-  /** Method to update this voltage range with line voltage delta.
+  /** Method to update this voltage range with voltage delta.
     *
     * @param deltaV
-    *   To consider.
+    *   The voltage difference to consider for the range limits.
     * @return
     *   A new [[VoltageRange]].
     */
-  def updateWithLineDelta(
-      deltaV: ComparableQuantity[Dimensionless]
+  def updateWithVoltageDelta(
+      deltaV: squants.Dimensionless
   ): VoltageRange = {
-    val (plus, minus) = (
-      deltaV.isGreaterThan(deltaPlus),
-      deltaV.isGreaterThan(deltaMinus),
-    ) match {
+
+    val (plus, minus) = (deltaV < deltaPlus, deltaV > deltaMinus) match {
       case (true, true) =>
-        (deltaPlus, deltaPlus)
-      case (false, true) =>
-        (deltaPlus, deltaV)
+        // between both range limits => set upper limit to deltaV, lower limit is unchanged
+        (deltaV, deltaMinus)
```

Comment:
Would this result in always tapping to the upper range, also if there is actually no congestion? No, or? Just double checking my guess.